src/app/collection/pages/collection-ncyte/feature/collection-learning-object-card/collection-learning-object-card.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'clark-collection-learning-object-card',
+  templateUrl: './collection-learning-object-card.component.html',
+  styleUrls: ['./collection-learning-object-card.component.scss']
+})
+export class CollectionLearningObjectCardComponent implements OnInit {
+  @Input() learnObj = new Input();
```

Comment:
You don't need to set this equal to a new Input() you can just type it as a learningObject.